LiteCore/Storage/KeyStore.cc

```diff
@@ -66,12 +66,13 @@ namespace litecore {
         return seq;
     }
 
-    bool KeyStore::createIndex(slice name,
-                               slice expressionJSON,
-                               IndexSpec::Type type,
-                               const IndexSpec::Options* options)
+    bool KeyStore::createIndex2(slice name,
```

Comment:
I think I'd prefer this be named just `createIndex`, since C++ allows overloading.

Reply:
Same for me!  This method already has some default parameters, the query language could be another so it doesn't even have to be overloaded if you add it at the end.  Overloading works too though.

Reply:
Changed accordingly.